ci-operator/config/openshift/knative-eventing/openshift-knative-eventing-release-v0.17.2__45.yaml

```diff
@@ -0,0 +1,280 @@
+base_images:
```

Comment:
@cardil  pleaes remove this file.

We have disable the 4.5 run for the 0.17+ branches.

Reply:
The script generated it. Should we update the script?

Reply:
https://github.com/openshift/knative-eventing/pull/950 should do the trick